contract_isp/contract.py

```diff
@@ -21,14 +21,19 @@
 ##############################################################################
 
 import logging
+_logger = logging.getLogger(__name__)
```

Comment:
I expect this to get flagged by flake8, for not having all the import statements at top.